__tests__/app.test.js

```diff
@@ -35,3 +35,59 @@ describe('get /api/topics', () => {
     });
     
 });
+
+describe('Name of the group', () => {
+    it('returns status 200 and an array of all articles objects with correct properties', () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body})=> {
+            expect(Array.isArray(body)).toBe(true);
+            expect(body).toBeSortedBy("created_at", {
+                descending: true,
+            });
```

Comment:
A .forEach can return a false positive if it is empty - how can we check that it is not empty?